packages/contracts/lib.esm/index.d.ts

```diff
@@ -18,23 +19,21 @@ export interface PayableOverrides extends Overrides {
     value?: BigNumberish | Promise<BigNumberish>;
 }
 export interface CallOverrides extends PayableOverrides {
-    blockTag?: BlockTag | Promise<BlockTag>;
     from?: string | Promise<string>;
 }
 export interface PopulatedTransaction {
-    to?: string;
-    from?: string;
-    nonce?: number;
+    to?: AccountLike;
+    from?: AccountLike;
     gasLimit?: BigNumber;
-    gasPrice?: BigNumber;
-    data?: string;
-    value?: BigNumber;
```

Comment:
Lets keep the same interface for `value` to be `BigNumber`. What is the reason behind changing it to `BigNumberish`?

Reply:
same here